pkg/build/admission/config.go

```diff
@@ -1,25 +1,35 @@
 package admission
 
 import (
-	"io"
 	"io/ioutil"
-	"reflect"
+	"os"
+
+	"github.com/golang/glog"
 
 	"k8s.io/kubernetes/pkg/runtime"
 
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	configlatest "github.com/openshift/origin/pkg/cmd/server/api/latest"
+	"github.com/openshift/origin/pkg/cmd/util/pluginconfig"
 )
 
 // ReadPluginConfig will read a plugin configuration object from a reader stream
-func ReadPluginConfig(reader io.Reader, config runtime.Object) error {
-	if reader == nil || reflect.ValueOf(reader).IsNil() {
+func ReadPluginConfig(pluginConfig map[string]configapi.AdmissionPluginConfig, name string, config runtime.Object) error {
+
+	configFilePath, err := pluginconfig.GetPluginConfigFile(pluginConfig, name, "")
+	if configFilePath == "" {
 		return nil
 	}
-
-	configBytes, err := ioutil.ReadAll(reader)
+	configData, err := os.Open(configFilePath)
 	if err != nil {
+		glog.Fatalf("Couldn't open plugin configuration %s: %#v", configFilePath, err)
 		return err
 	}
+
+	defer configData.Close()
+
+	configBytes, err := ioutil.ReadAll(configData)
+
 	err = configlatest.ReadYAMLInto(configBytes, config)
```

Comment:
thanks. i just moved this code from somewhere else but it makes sense to update it to use that instead.